openshift-knative-operator/pkg/monitoring/monitoring_test.go

```diff
@@ -0,0 +1,117 @@
+package monitoring
+
+import (
+	"testing"
+
+	mf "github.com/manifestival/manifestival"
+	"github.com/manifestival/manifestival/fake"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const servingNamespace = "knative-serving"
+
+func TestSetupServingRbacTrasfmormation(t *testing.T) {
+	client := fake.New()
```

Comment:
I could check the resources array from the manifest but using a client makes it more elegant.